### ISSUE TITLE
NomCom chair email announcements

### ISSUE THREAD

**Issue by ivandean** [ietf-tools/datatracker#8317]
### Description

NomCom chair has to be able to send email using the official email (nomcom-chair-yyyy@ietf.org) which will not be published automatically on the IETF NomCom announcement webpage. 
There are couple ways to do it
If NomCom chair selects Other email address, don't publish it automatically on the NomCom announcement webpage

If NomCom chair selects Other email address, add warning that the email will be published on the announcement webpage and offer option for the email NOT to be published on the announcement webpage.

### Code of Conduct

- [x] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
If we keep it so that the only way to send mail with that address in the From is to send from the datatracker, then we need to have two completely different workflows (that _look_ very different) to avoid accidentally invoking the workflow that wasn't intended. There would need to be a very distinct:

* Announce this message
vs
* Send this message privately

There are also concerns to address with how the datatracker _stores_ or doesn't store sent messages.

We could also try to work towards a way to send messages as the nomcom chair without going through the datatracker.

**Comment by richsalz** [comment]
I was a NomCom chair and I never needed this, how important is this?

**Comment by rjsparks** [comment]
Rich - when you sent slates to confirming boards, did you do so using your email client and with your personal (as in not nomcom-chair@" email as  the sender?

**Comment by richsalz** [comment]
Yes I used my work account. I also copied the appropriate liaison in case anyone was concerned about a forgery. I know Martin did the same thing, perhaps without copying the liaison. By that point in the calendar, everyone knew who the NomCom chair was, and there was no need to use an alternate From line.

**Comment by richsalz** [comment]
Also, when I received mail previously, of the "we did not pick you, thanks, let me know if you want to talk about it" kind of thing, it was also from the NomCom chair's personal address.

**Comment by ivandean** [comment]
I use gmail address and sometimes my email ends up in junk. I know that 
many ietf people white list ietf email domain, hence I would prefer to 
have option to send private email as nomcom-chair

**Comment by ivandean** [comment]
Sometimes using nomcom chair official email with the bodies for 
confirmation purposes, as well for sending out emails with the feedback. 
Using gmail account (as in my case, and many other people using the 
same), can lend the email into junk mail

**Comment by rjsparks** [comment]
Folding this into upcoming nomcom project work. We will support using the datatracker to send non-public mail as the nomcom chair. The primary use-case is for sending the confirmation messages to the confirming bodies.
